service/service.go

```diff
@@ -92,29 +92,57 @@ func (app *Application) Context() context.Context {
 	return context.Background()
 }
 
+// Parameters holds configuration for creating a new Application.
+type Parameters struct {
+	// ConfigFactory creates configuration.
```

Comment:
Would be nice to explain here that if configFactory is not provided the default factory will be used which will load from the file specified via command line option.